models/classification/_base_/resnet18.py

```diff
@@ -9,7 +9,7 @@
         style='pytorch'),
     neck=dict(type='GlobalAveragePooling'),
     head=dict(
-        type='LinearClsHead',
+        type='CustomLinearClsHead',
```

Comment:
Could you revert resnets? We are not using resnet for classification, but iseg can be affected by this change.